config/development.json

```diff
@@ -13,6 +13,7 @@
 		"store-alerts": true,
 		"unminified-js": true,
 		"wcpay": true,
-		"homescreen": true
+		"homescreen": true,
+		"mobile-app-banner": true
```

Comment:
I didn't see a js check for `window.wcAdminFeatures[ 'mobile-app-banner' ]` or the PHP feature flag check, so these can be removed if we don't plan on using them.

Reply:
@psealock this can't be removed because the Loader.php class loader depends on it to create feature plugins. There are a bunch of these that don't have a corresponding feature class either that get ignored, this could definitely be improved.

Reply:
The class factory for reference: https://github.com/woocommerce/woocommerce-admin/blob/main/src/Loader.php#L233

If you trace it back it loads the feature set from these json files.

Reply:
I'm wondering if we should add an array of features that have graduated to always being "on" so they don't have to be declared in the configs. Would you add that here or in a follow up?

Reply:
Sounds good. I think a follow up PR is ideal. Will ask you for some feedback on what list of features are always on.